tests/hashing/subtests/avoid-cascade/webpack/webpack.config.js

```diff
@@ -33,10 +33,22 @@ module.exports = {
     moduleIds: 'hashed',
     // create a dedicated bootstrap + hash mapping bundle:
     runtimeChunk: 'single',
-    // extract all shared dependencies from entry bundles:
     splitChunks: {
+      // extract all shared dependencies from entry bundles:
       chunks: 'all',
       minSize: 0,
+      cacheGroups: {
+        // change the caching behavior of assets:
+        assets: {
+          // Select assets:
+          test: /\.txt$/,
+          chunks: 'async',
+          // Target:
+          name: 'runtime',
+          // Do not check size limits:
+          enforce: true,
```

Comment:
Technically this is not needed here, as `minSize: 0` is already stated above, but to allow copy-pasting this into a real configuration (which usually don't have `minSize: 0`) I added that.

Reply:
Yeah, I'd like to remove `minSize: 0` from the webpack tests and instead pad files to get a particular result - webpack copies modules for a reason, and we shouldn't give the impression that it's 'good practice' to disable that.

We can note in the readme that webpack has thresholds for splitting, which is what I did in https://bundlers.tooling.report/code-splitting/multi-pages/.

Reply:
It might be helpful to have a `largeModule.js` lying around somewhere in the root of the repo

Reply:
I guess we could publish that to npm? I'd rather the tests were self-contained.